pkg/chartutil/files.go

```diff
@@ -24,6 +24,7 @@ import (
 
 	"github.com/ghodss/yaml"
 
+	"fmt"
```

Comment:
would you mind running `go fmt` on this? It should be at the top of the file, below line 21.

Reply:
Sure no problem